src/epub/text/chapter-19.xhtml

```diff
@@ -31,9 +31,9 @@
 			<p>“There has been many a craft,” the other said, “has sailed for many a port, and made this beach her last; and many an honest man has landed on it, and never one alive in such a sea. Nor would this one be living either, if it had not been for that other brave heart to stand by and save him. Brave heart, brave heart,” he said over to himself. “Here, pass me the bottle or I shall get the vapours. ’Tis good against these early chills, and I have not been in this place for ten years past, since poor Elzevir was cut adrift.”</p>
 			<p>I could not see the speaker’s face from where I lay upon the floor, yet seemed to know his voice; and so was fumbling in my weakened mind to put a name to it, when he spoke of Elzevir, and sent my thoughts flying elsewhere.</p>
 			<p>“Elzevir,” I said, “where is Elzevir?” and sat up to look round, expecting to see him lying near me, and remembering the wreck more clearly now, and how he had saved me with that last shove forward on the beach. But he was not to be seen, and so I guessed that his great strength had brought him round quicker than had my youth, and that he was gone back to the beach.</p>
-			<p>“Hush,” said one of the men at the table, “lie down and get to sleep again”; and then he added, speaking to his comrade: “His brain is wandering yet: do you see how he has caught up my words about Elzevir?”</p>
-			<p>“No,” I struck in, “my head is clear enough; I am speaking of Elzevir Block. I pray you tell me where he is. Is he well again?” They got up and stared at one another and at me, when I named Elzevir Block, and then I knew the one that spoke for Master Ratsey only greyer than he was.</p>
-			<p>“Who are you?” he cried, “who talk of Elzevir Block.”</p>
```

Comment:
I think `cried` is correct here, to match the tense of the rest of the chapter. If it's like this in print then it could be a printer error.